README.rst

```diff
@@ -1,13 +1,19 @@
-A Go kernel for IPython.
+A Go kernel for IPython
+=======================
 
 To install::
 
+    pip install 'ipython>=3.0.0' jinja2 jsonschema pyzmq tornado werkzeug
```

Comment:
We recommend doing `pip install ipython[notebook]` - that will get the notebook dependencies automatically.